django/system/hsp-django-cve-2019_19844.yaml

```diff
@@ -0,0 +1,17 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorHostPolicy
+metadata:
+  name: hsp-django-cve-2019_19844
+spec:
+  tags: ["Django", "CVE", "VM", "2019_19844", "Password-Reset"]
+  message: "Alert Password Reset Function is used"
+  file:
+    severity: 2
+    matchPaths:
+      - path: /django/accounts/urls.py
```

Comment:
Can you attach some ref for this, I want to validate this file is directly linked to password reset

Reply:
@raviknox This is the Django VM policy for the blog . I have mentioned everything here in this document . Please kindly refer this [https://docs.google.com/document/d/1fnerXXOsMyXfym3wSGZjEXnkpMsJxIXqDCKbkp5S8RA/edit#heading=h.5a3k5d65sfp7](url)